docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -525,8 +525,128 @@ as follows:
 	meta-analysis for the effect size causing a slightly non-symmetrical 
 	confidence interval around the mean.
 
-Determining Whether A Simulant is Affected
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**Birth Weight**
+
+The effect of maternal consumption of iron fortified food on infant birth 
+weight was obtained from Haider et al. (2013). According to this data source,  
+birth weight among babies born to mothers increased, on average, by 15.1 grams 
+(95% CI: 6.0, 24.2) per 10 mg of daily iron consumption. The distribution of 
+the parameter should be modeled as follows:
+
+.. code-block:: Python
+
+	from scipy.stats import norm
+
+	# mean and 0.975-quantile of normal distribution for mean difference (MD)
+	mean = 15.1
+	q_975 = 24.2
+
+	# 0.975-quantile of standard normal distribution (=1.96, approximately)
+	q_975_stdnorm = norm().ppf(0.975)
+
+	std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+	# Frozen normal distribution for MD, representing uncertainty in our effect size
+	bw_md_distribution = norm(mean, std)
+
+	# random sample from effect size distribution
+	bw_md_per_10_mg_iron = bw_md_distribution.rvs()
+
+.. note::
+
+	The Haider et al. 2013 paper did not report if the dose of iron was 
+	measured as elemental iron or as an iron compound such as NaFeEDTA. We 
+	operated under the assumption that 10 mg of daily iron consumption, as 
+	referenced in the Haider paper, represented 10 mg of *elemental* iron.
+
+Therefore, we investigated the amount of daily iron consumption that a 
+pregnant mother would likely consume through iron fortification of staple 
+foods in our locations of interest in order to scale the effect size 
+accordingly. See the table below:
+
+.. list-table:: Daily Iron Consumption Parameters
+  :widths: 5 5 5
+  :header-rows: 1
+
+  * - Location
+    - Concentration of forticant in flour
+    - Amount of fortifiable flour consumed daily
+  * - Ethiopia
+    - 30 mg NaFeEDTA / kg flour  [1]
+    - 100 g (IQR: 77.5, 200) [2]
+  * - India
+    - 14 to 21.25 mg NaFeEDTA / kg flour [1]; assume midpoint value of **17.6**
+    - 100 g (IQR: 77.5, 200) [2*]
+  * - Nigeria
+    - 40 mg NaFeEDTA / kg flour [1]
+    - 100 g (IQR:77.5, 200) [2*]
+
+.. note::
+
+	While there was data for the concentration of iron forticants other than 
+	NaFeEDTA for the locations in this table, we conservatively chose to  use 
+	the concentrations of NaFeEDTA because elemental iron is uesd at a *lower* 
```

Comment:
While there was data for the concentration of iron forticants other than NaFeEDTA for the locations in this table, we conservatively chose to use the concentrations of the NaFeEDTA forticant. This is because the concentration of elemental iron in flour is lower when the NaFeEDTA forticant is used compared to other forticants.